spring-cloud-gcp-storage/pom.xml

```diff
@@ -28,5 +28,10 @@
 			<artifactId>spring-integration-file</artifactId>
 			<optional>true</optional>
 		</dependency>
-	</dependencies>
+    <dependency>
```

Comment:
Wrong indents for this new dependency definition.
We should use `tabs` everywhere unless it is ASCIIDoc

Reply:
fixed! 

About that SpringBootTest annotation it was mistakenly used when just ContextConfig was necessary. I fixed the other location where I copied it from.